### PR TITLE
Close CUBE beta

### PR DESCRIPTION
**Pull request by mrgnr**
## Done

- Update copy on /cube to close the beta
- Redirect /cube/microcerts and /cube/study to /cube
- Update the CUBE contact form

## QA

- Open the demo at https://ubuntu-com-11224.demos.haus/cube
  - Verify that the content is correct according to https://github.com/canonical-web-and-design/commercial-squad/issues/491
- Click the "Sign up" button and verify that the form content is correct according to https://github.com/canonical-web-and-design/commercial-squad/issues/491
- Go to https://ubuntu-com-11224.demos.haus/cube/microcerts and verify that you are redirected to https://ubuntu-com-11224.demos.haus/cube
- Go to https://ubuntu-com-11224.demos.haus/cube/study and verify that you are redirected to https://ubuntu-com-11224.demos.haus/cube

## Issue

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/491 // [[Jira](https://warthogs.atlassian.net/jira/software/projects/CUBE/boards/490?assignee=613a0f86ee2fd0006ae0c8bf&selectedIssue=CUBE-84)]

### CODE REVIEW

**Comment by wgx** [comment]
Looks good to me 👍🏽

**Review by wgx — APPROVED** [review]
LGTM

**Comment by lex-joy** [comment]
I'm unable to open the issue link under "QA," but I'd recommend that we change the copy text slightly (because there is some American English in the original that I missed the first time):

> <h2>The beta is now closed</h2>
> <p>Thank you for your interest in Project CUBE. This beta is now closed.</p>
> <p>As a result of the data gathered, we will be able to expand our certification programme to include offerings that better meet the needs of our community.</p>
> <p>If you would like to be a part of this exciting new phase of our programme, please click the button below to:
> <li>Be considered for forthcoming beta exams.</li>
> <li>Participate in our research efforts, including focus groups and questionnaires.</li>
> <li>Sign up to receive email updates.</li>
> </p>

**Comment by mrgnr** [comment]
@lex-joy I updated the copy and removed the job role field from the form.

**Comment by linux-in-flux** [comment]
A few notes from Thibaut:
I've updated the copy docs page for /cube to reflect his comments.
As for the form, I was unable to find a "copy doc" for this, but it's a rather extensive system and I may have missed it. He said that it is standard practice now for web forms that require scrolling to have a next button to go to the second page, and he wanted to list of microcerts to be in columns to reduce scrolling. As for me, one of the original changes to the checkboxes at the bottom wasn't made.

Images included for reference (2nd and 3rd image should be on one page, but had to be edited separately):

revisedbetapage1
revisedbetapage2part1
revisedbetapage2part2

**Comment by mrgnr** [comment]
@linux-in-flux I updated the copy on /cube (I think it looks much better overall), formatted the microcerts interest section into two columns, and fixed the research checkbox text (this was my bad).

For QA, have a look at and verify that everything is correct:
https://ubuntu-com-11224.demos.haus/cube
https://ubuntu-com-11224.demos.haus/cube#get-in-touch
https://ubuntu-com-11224.demos.haus/cube/contact-us

As for the multi-page form, this is non-trivial to implement because we will need to store the form state as the user progresses through the steps. We can certainly add that, but it will take a bit more work. (Any thoughts on this @wgx ?)

**Comment by linux-in-flux** [comment]
The site and form look good to me. We can talk about multipage in our meeting tomorrow. I may be able to avoid or delay this change if it is particularly onerous.

File: webapp/shop/cube/views.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def cube_microcerts(
     """
     View for Microcerts homepage
     """
+
+    # Temporarily redirect to /cube while the beta is closed
+    return flask.redirect("/cube", code=302)
```
Comment (jpmartinspt) [inline webapp/shop/cube/views.py:39]: I think you could just use `redirects.yaml` for this stuff. The check is always done before each request is handled so I think you could actually just leave the views as they are.

Comment (mrgnr) [reply webapp/shop/cube/views.py:39]: Thanks, I wasn't aware of `redirects.yaml`. I've made the change and it seems to work fine.

File: navigation.yaml [hunk 2]
```diff
@@ -1809,12 +1809,13 @@ cube:
...
-    - title: Study
-      path: /cube/study
-      persist: True
+    # Temporarily disable for closing the beta
+    # - title: Microcerts
+    #  path: /cube/microcerts
+    #
+    # - title: Study
+    #  path: /cube/study
+    #  persist: True
```
Comment (ClementChaumel) [inline navigation.yaml:1818]: I wonder if we should just remove it altogether instead. We can always look into git's history to restore it when we need.

File: templates/cube/index.html [hunk 3]
```diff
@@ -9,13 +9,16 @@
...
-    <div class="col-6">
-      <h1>You&rsquo;re a professional.</br>Prove it with CUBE.</h1>
+    <div class="col-12">
+      <h1>You&rsquo;re an Ubuntu professional.<br>Prove it with certification.</h1>
+      <!-- Temporarily disable for closing the beta
       <p class="p-heading--4">The Certified Ubuntu Engineer (CUBE) is a professional endorsement indicating mastery of Ubuntu. You need to pass 15 microcerts to earn your CUBE qualification.</p>
       <p>
         <a href="/cube/microcerts{% if get_test_backend %}?test_backend=true{% endif %}" class="p-button--positive">View the microcerts</a>
       </p>
+      -->
```
Comment (ClementChaumel) [inline templates/cube/index.html:19]: Same here

**Review by ClementChaumel — APPROVED** [review]
Code +1, just not sure if we should keep the beta content commented out as we don't know for how long it will be disabled and it might be subject to change when we reintroduce it anyway.

**Comment by wgx** [comment]
@mrgnr No UX issues, as long as its clear its multipart and its clear to users they are progressing.

**Comment by linux-in-flux** [comment]
LGTM with caveat that we'll go back in and paginate form later.

File: navigation.yaml (hunk 2 above)
Comment (mrgnr) [reply navigation.yaml]: I went ahead and removed all the commented-out code.

**Review by jpmartinspt — APPROVED** [review]

**Pull request merged**